conftest.py

```diff
@@ -85,27 +85,8 @@ def base_url(env):
     return TestData.BASE_URL[env]
 
 
-@pytest.fixture(scope="session")
-def browser_from_cmd(request):
-    """
-    Work in progress
-    E.g.: chrome, firefox, all.
-    :param request:
-    :return: The list of browsers on which to perform the tests. For running on multiple browsers.
-    """
-    browser_opt = request.config.getoption("browser").lower()
-    if browser_opt == "all":
-        browsers = ["chrome", "firefox"]
```

Comment:
de ce ai sters acest fixture?

Reply:
pt ca nu era folosit, si am renuntat sa mai avem optiunea de --browser all in terminal.